gemrb/core/Map.h

```diff
@@ -529,6 +529,17 @@ class GEM_EXPORT Map : public Scriptable {
 	PathMapFlags GetBlocked(const NavmapPoint&) const;
 	PathMapFlags GetBlocked(const NavmapPoint&, int size) const;
 	PathMapFlags GetBlockedTile(const SearchmapPoint&) const;
+
+	// helper function used when the size > 2
+	PathMapFlags GetChildBlockedStatusForBigSize(const SearchmapPoint& smptChild, const unsigned int size) const
+	{
+		return GetBlockedInRadiusTile(smptChild, size);
+	}
+	// helper function used when the size <= 2
+	PathMapFlags GetChildBlockedStatusForSmallSize(const SearchmapPoint& smptChild, const unsigned int /* size */) const
+	{
+		return GetBlockedTile(smptChild);
+	}
```

Comment:
Do you plan to use these anywhere else? Otherwise I suggest you add them as lambdas instead.